ioi-2022.html

```diff
@@ -212,10 +212,10 @@
     </p>
 
 modal_link_text: |
-    <h2 class="collect-explanation-title">¿Para qué necesitamos el dinero?</h2>
+   
 
 event_info: |
-    <div>IOI 2022 - Indonesia</div>
+    <div>EGOI 2022 - Turquía</div>
     <div>International Olympiad in Informatics</div>
```

Comment:
Usa el estilo de CSS `word-wrap: break-word` para que cuando la línea haga wrap (se parta en 2), no se parta en medio de una palabra, que es lo que sucede en tu screenshot, de la palabra "in" es partida en dos